klimaat_helpdesk/templates/core/question_submitted.html

```diff
@@ -45,7 +45,7 @@ <h1 class="form__header">{% trans 'Bedankt voor je vraag!' %}</h1>
             {{ form.accept_terms.label_tag }}
             {{ form.accept_terms }}
             <p>
-              <a href="/terms" target="_blank">{% trans "Our terms and conditions" %}</a>
+              <a href="/terms" target="_blank">{% trans 'Onze voorwaarden' %}</a>
```

Comment:
Should it be /terms or /voorwaarden ?

Reply:
i am still using /terms. I don't get the obsession with slugs been dutch or english. makes translation of the website only more difficult!
so no changes here